docs/tutorial/import_export.rst

```diff
@@ -0,0 +1,1021 @@
+want to name different reaction files based on the subsystem designations or
+cellular compartments, or if you want to separate the temporary reactions
+that are used to fill reaction gaps from the main model. An example of how you
+could designate multiple reaction files is found below. This file can be found
+in the additional files folder in the file ``complex_model.yaml``.
+
+.. code-block:: yaml
+
+    name: Ecoli_core_model
+    biomass: Biomass_Ecoli_core_w_GAM
+    default_flux_limit: 1000.0
+    model:
+    - include: core_model_definition.tsv
+    compounds:
+    - include: compounds.yaml
+    reactions:
+    - include: reactions/cytoplasm.yaml
+    - include: reactions/periplasm.yaml
+    - include: reactions/transporters.yaml
+    - include: reactions/extracellular.yaml
+    media:
+    - include: medium.yaml
+    limits:
+    - include: limits.yaml
+
+
+As can be see here the modeler chose to distribute their reaction database
+files into different files representing various cellular compartments and roles.
+This organization can be customized to suit your preferred workflow.
+
+There are also situations where you may wish to designate only a subset
+of the reaction database in a metabolic simulation. In these situations it is
+possible to use a model definition file to identify which subset of reactions
+will be used from the larger database. The model definition file is simply a
+list of reaction IDs that will be included in the simulation.
+
+An example of how to include a model definition file can be found below.
+
+.. code-block:: yaml
+
+    name: Ecoli_core_model
+    biomass: Biomass_Ecoli_core_w_GAM
+    default_flux_limit: 1000.0
+    model:
+    - include: subset.tsv
+    compounds:
+    - include: compounds.yaml
+    reactions:
+    - include: reactions.yaml
+    media:
+    - include: medium.yaml
+    limits:
+    - include: limits.yaml
+
+.. note::
+    When the model definition file is not identified, PSAMM will include
+    the entire reaction database in the model. However, when it is identified,
+    PSAMM will only include the reactions that are listed in the model
+    definition file in the model. This design can be useful when you want to
+    make targeted tests on a subset of the model or when you want to include a
+    larger database for use with the gap filling functions.
+
+Reactions
+~~~~~~~~~
+
+The ``reactions.yaml`` file is where the reaction information is stored in the
+model. A sample from this file can be seen below:
+
+.. code-block:: yaml
+
+    - id: ACALD
+      name: acetaldehyde dehydrogenase (acetylating)
+      genes: b0351 or b1241
+      equation: '|acald_c[c]| + |coa_c[c]| + |nad_c[c]| <=> |accoa_c[c]| + |h_c[c]| +
+        |nadh_c[c]|'
+      subsystem: Pyruvate Metabolism
+    - id: ACALDt
+      name: acetaldehyde reversible transport
+      genes: s0001
+      equation: '|acald_e[e]| <=> |acald_c[c]|'
+      subsystem: Transport, Extracellular
```

Comment:
I think the import would look slightly different when run with a recent version. The `_c` suffixes would probably be removed from reactions, the `|` would only be used for compounds with space characters in the ID (i.e. almost never), and the quoting would no longer be necessary because the reaction equation no longer starts with the `|` character. It may be a good idea to do a new reimport so make sure that the output here is accurate?